crates/language_models/src/provider/vercel.rs

```diff
@@ -198,35 +198,33 @@ impl LanguageModelProvider for VercelLanguageModelProvider {
     }
 
     fn provided_models(&self, cx: &App) -> Vec<Arc<dyn LanguageModel>> {
-        let mut models = BTreeMap::default();
+        let mut models = Vec::new();
```

Comment:
moved to vec as btree was automatically sorting the models and they were getting jumbled. Expected order should be Vercel v0 1.5 Small,Vercel v0 1.5 Medium,Vercel v0 1.5 Large but it was sorting it alphabetically and making Vercel v0 1.5 Large first and Small last.